demo/px-vis-pie-custom-component.html

```diff
@@ -0,0 +1,148 @@
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../px-vis/px-vis-pie.html">
+<link rel="import" href="../../px-vis/px-vis-svg.html">
+
+<link rel="import" href="../../px-vis/px-vis-behavior-common.html">
+
+
+<dom-module id="px-vis-pie-custom-component">
+    <template>
```

Comment:
This "custom" component was extracted from the px-vis/demo/px-vis-pie-demo-component.html, I did adjust the properties such as height and radius so that it can fit inside the thumbnail slot nicely. I didn't use px-vis-pie-chart since I thought it is simpler to use px-vis-pie-demo-component.